src/EA.Weee.Web/ViewModels/Shared/Mapping/ViewEvidenceNoteViewModelMap.cs

```diff
@@ -106,18 +106,19 @@ private void SetSuccessMessage(EvidenceNoteData note, object noteStatus, ViewEvi
                     switch (status)
                     {
                         case NoteStatus.Submitted:
-                            model.SuccessMessage =
```

Comment:
there should be  unit tests for that class - ViewEvidenceNoteViewModelMapTests,

have you updated the tests which check for the message text?